pyproject.toml

```diff
@@ -66,13 +65,15 @@ rev_comp_se_primers = "mgnify_pipelines_toolkit.analysis.amplicon.rev_comp_se_pr
 standard_primer_matching = "mgnify_pipelines_toolkit.analysis.amplicon.standard_primer_matching:main"
 mapseq_to_asv_table = "mgnify_pipelines_toolkit.analysis.amplicon.mapseq_to_asv_table:main"
 primer_val_classification = "mgnify_pipelines_toolkit.analysis.amplicon.primer_val_classification:main"
+amplicon_study_summary_generator = "mgnify_pipelines_toolkit.analysis.amplicon.study_summary_generator:cli"
```

Comment:
@SandyRogers we need to make sure that rename is reflected in prefect flow :mind-panic:

Reply:
Yep good point! That should be okay as toolkit is version-pinned in automation requirements.. and I think tests will fail if we upgrade it and forget to change :D